internal/services/network/web_application_firewall_policy_resource.go

```diff
@@ -1296,3 +1296,57 @@ func flattenWebApplicationFirewallPoliciesSubResourcesToIDs(input *[]webapplicat
 
 	return ids
 }
+
+func convertRuleGroupNameToBeAcceptedByAPI(name string) string {
+	var ruleGroupName string
+	switch name {
+	case "APPLICATION-ATTACK-LFI":
+		ruleGroupName = "LFI"
+	case "APPLICATION-ATTACK-RFI":
+		ruleGroupName = "RFI"
+	case "APPLICATION-ATTACK-RCE":
+		ruleGroupName = "RCE"
+	case "APPLICATION-ATTACK-PHP":
+		ruleGroupName = "PHP"
+	case "APPLICATION-ATTACK-NodeJS":
+		ruleGroupName = "NODEJS"
+	case "APPLICATION-ATTACK-XSS":
+		ruleGroupName = "XSS"
+	case "APPLICATION-ATTACK-SQLI":
+		ruleGroupName = "SQLI"
+	case "APPLICATION-ATTACK-SESSION-FIXATION":
+		ruleGroupName = "FIX"
+	case "APPLICATION-ATTACK-SESSION-JAVA":
+		ruleGroupName = "JAVA"
+	default:
+		ruleGroupName = name
+	}
+	return ruleGroupName
+}
+
+func convertRuleGroupNameFromAPI(name string) string {
+	var ruleGroupName string
+	switch name {
+	case "LFI":
+		ruleGroupName = "APPLICATION-ATTACK-LFI"
+	case "RFI":
+		ruleGroupName = "APPLICATION-ATTACK-RFI"
+	case "RCE":
+		ruleGroupName = "APPLICATION-ATTACK-RCE"
+	case "PHP":
+		ruleGroupName = "APPLICATION-ATTACK-PHP"
+	case "NODEJS":
+		ruleGroupName = "APPLICATION-ATTACK-NodeJS"
+	case "XSS":
+		ruleGroupName = "APPLICATION-ATTACK-XSS"
+	case "SQLI":
+		ruleGroupName = "APPLICATION-ATTACK-SQLI"
+	case "FIX":
+		ruleGroupName = "APPLICATION-ATTACK-SESSION-FIXATION"
+	case "JAVA":
+		ruleGroupName = "APPLICATION-ATTACK-SESSION-JAVA"
+	default:
+		ruleGroupName = name
+	}
+	return ruleGroupName
+}
```

Comment:
Then we can also remove this unnecessary conversion.

Reply:
fixed.